tests/test_doctestplus.py

```diff
@@ -731,4 +732,38 @@ def test_remote_data(testdir):
     )
     testdir.inline_run(p, '--doctest-plus', '--doctest-rst').assertoutcome(skipped=1)
 
+    p = testdir.makefile(
+        '.rst',
+        """
+        
+        #This test is skipped when remote data is not requested, as it should
+        #but when remote data is requested the +FLOAT_CMP directive fails
+        .. doctest-remote-data::
+            >>> x = 1/3.
+            >>> x    # doctest: +FLOAT_CMP
+            0.333333
+            #
+            # do some failing to check whether the whole block is indeed skipped
+            >>> 1 + 1
+            3
```

Comment:
@saimn @pllim - we're kind of stuck here. This code chunck is skipped as expected with the new directive, but the test comes back as "passed" rather than "skipped". Have you recall any similar situation, or the reason for this?

Reply:
I mean the pass instead of skipped status is there when this is run without `--remote-data`, with the one below, it should not be skipped but fail status, so please only